src/requirements.txt

```diff
@@ -4,3 +4,4 @@ ndjson ~=0.3
 pytz >=2021.1
 us ~=2.0
 shapely ~=1.7
+tqdm ~=4.62
```

Comment:
Love it!

Reply:
Ha, I just realized this is still here, but I don't think I actually used it in the end! It was real nice before I switched to multiprocessing, though.

Guess I should either use it to display how many files have been read or drop it.